plugins/traction_innkeeper/traction_innkeeper/v1_0/oca/oca_service.py

```diff
@@ -127,7 +127,7 @@ async def list_oca_records(
         is_root_profile = issuer_profile == self.profile
         public_info = await self.get_public_did_info(issuer_profile)
         tag_filter = self.build_tag_filter(schema_id, cred_def_id)
-        post_filter = self.build_post_filter(public_info)
+        post_filter = {} if is_root_profile == True else self.build_post_filter(public_info)
         records = []
         if is_root_profile or public_info:
```

Comment:
kind of inconsequential, but could change this logic.

Reply:
Oh yeah thanks for catching, left `== True` in there while debugging